ledger-tool/src/output.rs

```diff
@@ -939,6 +939,29 @@ impl serde::Serialize for AccountsScanner {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct CliAccounts {
```

Comment:
It feels a little awkward having `CliAccounts` live here and `CliAccount` in `cli-output`, but I guess this makes sense if it's only used by `ledger-tool`